<launch>
<param name="video_source" value="$(find windMill)/../../../../samples/top/topBlue.MOV"/>

<!--<param name="video_source" value="$(find windMill)/../../../../samples/windmill/dou.avi"/>-->
<param name="dbg_img_path" value="$(find windMill)/../../../../dbg_img/"/>
<param name="is_video_dbg" value="1"/>
    <node pkg="windMill" type="windMill_node" name="windMill" output="screen" >
        <remap from="/MVCamera/image_raw" to ="/camera/image_raw"/>
     </node>
    <node pkg="video_pub" type="video_pub_node" name="video_pub" output="screen" respawn="true" />
     <node pkg="img_displayer" type="img_displayer_node" name="img_displayer" output="screen"  />
<!--    <node pkg="numPred" type="numPred_node.py" name="numPred" output="screen" />-->
    <node pkg="serial_common" type="serial_common_node" name="serial_common_node" output="screen" >
    <remap from="/write" to ="/write_pixel"/>
    </node>

<rosparam command="load" file="$(find windMill)/cfg/params.yaml" />

</launch>
